pandas/core/arrays/interval.py

```diff
@@ -675,7 +677,8 @@ def _shallow_copy(self: IntervalArrayT, left, right) -> IntervalArrayT:
         """
         dtype = IntervalDtype(left.dtype, closed=self.closed)
         left, right, dtype = self._ensure_simple_new_inputs(left, right, dtype=dtype)
-        self._validate(left, right, dtype=dtype)
+        if verify_integrity:
```

Comment:
so is this still necessary?

Reply:
Not for this regression, but I think we should look through all usages to see if it's necessary everywhere